src/AbstractDoctrineAnnotationFixer.php

```diff
@@ -215,6 +219,10 @@ private function nextElementAcceptsDoctrineAnnotations(PhpTokens $tokens, $index
 
         while ($tokens[$index]->isGivenKind(array(T_PUBLIC, T_PROTECTED, T_PRIVATE, T_FINAL, T_ABSTRACT))) {
             $index = $tokens->getNextMeaningfulToken($index);
+
+            if (null === $index) {
```

Comment:
Not sure this can ever happen: no meaningful token after `public`, `protected`, `private`, `final` or `abstract` would be a parse error.

Reply:
I added this second change only to be safe, but not covered by any test (this specific addition).

Reply:
Sorry for that, done.